server_admin/topics/user-federation/ldap.adoc

```diff
@@ -167,5 +167,9 @@ When the password of user is updated from {project_name} and sent to LDAP, it is
 updating the password to built-in {project_name} database, when the hashing and salting is applied to the password before it is sent to DB.
 In the case of LDAP, the {project_name} relies on the LDAP server to provide hashing and salting of passwords.
 
-Most of LDAP servers (Microsoft Active Directory, RHDS, FreeIPA) provide this by default. Some others (OpenLDAP, ApacheDS) may store the passwords
-in plain-text by default and you may need to explicitly enable password hashing for them. See the documentation of your LDAP server more details.
+Most LDAP servers (Microsoft Active Directory, RHDS, FreeIPA) provide this by default. Some others (OpenLDAP, ApacheDS) may store the passwords
```

Comment:
change the parenthetical comments to "such as...
For example, "Most LDAP servers, such as Microsoft Active Directory, RHDS, and FreeIPA, provide...

Reply:
Done.